packages/grid/x-data-grid-pro/tsconfig.json

```diff
@@ -1,9 +1,7 @@
 {
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
-    "noEmit": true,
-    "allowJs": true,
-    "types": ["mocha", "node", "chai", "chai-dom"]
+    "types": ["react", "mocha", "node"]
```

Comment:
`yarn typescript` fails on my machine:

```sh
$ yarn typescript
yarn run v1.22.17
$ lerna run --no-bail --parallel typescript
lerna notice cli v4.0.0
lerna info Executing command in 6 packages: "yarn run typescript"
docs: $ tsc -p tsconfig.json
@mui/x-data-grid: $ tsc -p tsconfig.json
@mui/x-data-grid-generator: $ tsc -p tsconfig.json
@mui/x-data-grid-pro: $ tsc -p tsconfig.json
storybook: $ tsc -p tsconfig.json
@mui/x-license-pro: $ tsc -p tsconfig.json
@mui/x-data-grid: src/internals/utils/createSelector.test.ts(12,37): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/internals/utils/createSelector.test.ts(15,44): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/cells.DataGrid.test.tsx(38,37): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/cells.DataGrid.test.tsx(50,37): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/cells.DataGrid.test.tsx(64,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/cells.DataGrid.test.tsx(77,35): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/cells.DataGrid.test.tsx(96,35): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/cells.DataGrid.test.tsx(120,8): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/columnHeaders.DataGrid.test.tsx(37,46): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/columnHeaders.DataGrid.test.tsx(49,46): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/components.DataGrid.test.tsx(61,37): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/components.DataGrid.test.tsx(75,33): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/components.DataGrid.test.tsx(127,8): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/components.DataGrid.test.tsx(148,12): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/filtering.DataGrid.test.tsx(61,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/keyboard.DataGrid.test.tsx(508,25): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(119,59): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(128,50): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(175,12): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(214,12): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(228,12): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(250,12): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(308,23): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(344,23): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(378,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(442,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(474,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(538,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(564,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(604,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(617,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(658,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(667,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(686,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(839,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(863,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/layout.DataGrid.test.tsx(972,38): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/pagination.DataGrid.test.tsx(344,10): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/pagination.DataGrid.test.tsx(354,10): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/pagination.DataGrid.test.tsx(362,10): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/pagination.DataGrid.test.tsx(518,72): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/pagination.DataGrid.test.tsx(520,72): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/rows.DataGrid.test.tsx(108,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/rows.DataGrid.test.tsx(109,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/rows.DataGrid.test.tsx(110,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/rows.DataGrid.test.tsx(142,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(419,32): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(420,36): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(421,34): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(425,34): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(428,34): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(430,32): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(431,36): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(513,69): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(519,69): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(536,14): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(700,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/selection.DataGrid.test.tsx(712,14): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/sorting.DataGrid.test.tsx(495,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(60,45): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(64,46): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(86,45): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(90,46): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(103,45): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(107,46): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(120,45): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(124,46): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid: src/tests/toolbar.DataGrid.test.tsx(254,22): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid: error Command failed with exit code 2.
@mui/x-data-grid: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(72,36): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(74,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(77,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(85,36): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(87,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(96,36): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(99,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(109,33): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(115,31): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(116,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(125,33): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(131,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(140,33): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(162,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(197,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(226,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(227,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(236,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(237,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(241,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(242,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(251,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(252,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(265,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(266,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(280,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(285,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(295,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(296,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(305,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(308,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(309,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(310,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(319,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(322,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(323,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(333,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(344,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(345,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(361,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(362,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(363,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(378,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(379,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(380,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(398,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(399,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(400,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(418,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(419,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(420,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(441,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(442,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(463,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(464,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(483,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(484,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(505,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(506,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(553,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(555,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(587,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(589,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(591,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(607,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(609,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(612,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(633,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(634,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(635,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(659,41): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(756,31): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(761,29): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(762,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(781,31): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(786,29): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(787,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(856,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(899,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(902,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/cellEditing.DataGridPro.test.tsx(904,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnHeaders.DataGridPro.test.tsx(170,53): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnHeaders.DataGridPro.test.tsx(173,53): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnHeaders.DataGridPro.test.tsx(175,57): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(77,75): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(78,76): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(79,74): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(82,71): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(83,72): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(84,70): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(94,71): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(95,72): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(96,70): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(98,75): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(99,76): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(100,74): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(112,24): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(119,24): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(131,32): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(137,32): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(147,26): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(154,26): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(155,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(165,26): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(172,26): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(173,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(183,55): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columnPinning.DataGridPro.test.tsx(184,55): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(101,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(102,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(122,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(123,29): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(169,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(170,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(174,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(175,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(186,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(187,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(197,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(198,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(209,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(210,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(214,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(215,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(226,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(227,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(231,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(232,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(243,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(244,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(254,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(255,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(266,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(267,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(277,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(278,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(289,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(290,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(294,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(295,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(306,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(307,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(317,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(318,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(329,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(339,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(345,40): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/columns.DataGridPro.test.tsx(390,8): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/components.DataGridPro.test.tsx(37,72): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/detailPanel.DataGridPro.test.tsx(36,23): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/detailPanel.DataGridPro.test.tsx(45,65): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/detailPanel.DataGridPro.test.tsx(47,23): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/detailPanel.DataGridPro.test.tsx(95,39): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/detailPanel.DataGridPro.test.tsx(99,39): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/detailPanel.DataGridPro.test.tsx(362,31): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(119,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(120,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(167,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(168,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(200,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(201,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(248,34): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(249,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(283,35): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(284,36): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(312,60): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(339,31): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(363,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(367,41): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(390,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(394,41): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(433,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(464,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(540,63): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(631,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(682,63): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(825,44): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/editComponents.DataGridPro.test.tsx(855,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/events.DataGridPro.test.tsx(219,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/events.DataGridPro.test.tsx(235,30): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/export.DataGridPro.test.tsx(644,10): error TS2339: Property 'toWarnDev' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(61,55): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(62,55): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(77,59): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(125,27): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(131,27): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(184,27): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(190,27): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/layout.DataGridPro.test.tsx(224,38): error TS2339: Property 'toHaveComputedStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/reorder.DataGridPro.test.tsx(319,31): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/reorder.DataGridPro.test.tsx(320,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(67,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(69,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(74,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(79,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(89,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(90,31): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(91,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(92,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(93,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(94,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(106,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(119,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(132,29): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(148,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(149,28): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(157,50): error TS2339: Property 'toHaveFocus' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(221,34): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(222,35): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(245,34): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(246,35): error TS2339: Property 'attribute' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(274,30): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(275,26): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(368,26): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowEditing.DataGridPro.test.tsx(432,35): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(742,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(746,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(753,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(1017,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(1018,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(1022,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(1023,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(1030,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rowGrouping.DataGridPro.test.tsx(1031,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(106,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(107,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(111,28): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(112,32): error TS2339: Property 'class' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(398,32): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(444,32): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(448,32): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(461,35): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(466,35): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(488,34): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(509,34): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/rows.DataGridPro.test.tsx(526,34): error TS2339: Property 'text' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/selection.DataGridPro.test.tsx(122,41): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/selection.DataGridPro.test.tsx(132,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/selection.DataGridPro.test.tsx(198,41): error TS2339: Property 'attr' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/treeData.DataGridPro.test.tsx(398,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/treeData.DataGridPro.test.tsx(400,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/treeData.DataGridPro.test.tsx(407,38): error TS2339: Property 'toHaveInlineStyle' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: src/tests/treeData.DataGridPro.test.tsx(509,10): error TS2339: Property 'toErrorDev' does not exist on type 'Assertion'.
@mui/x-data-grid-pro: error Command failed with exit code 2.
@mui/x-data-grid-pro: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! Received non-zero exit code 2 during execution
lerna success run Ran npm script 'typescript' in 6 packages in 15.4s:
lerna success - docs
lerna success - storybook
lerna success - @mui/x-license-pro
lerna success - @mui/x-data-grid-generator
lerna success - @mui/x-data-grid-pro
lerna success - @mui/x-data-grid
error Command failed with exit code 2
```

Reply:
I think it's caused by your `  "exclude": ["**/.*/**/*", "**/build/**/*", "**/node_modules/**/*", "docs/export/**/*"],`

Reply:
Right, we probably need to include `@mui/monorepo`, but exclude all other `node_modules`